packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -58,10 +58,11 @@ const DropdownMenu = props => {
 
   const renderTrigger = () => {
     const triggerComponent = props.renderTrigger(getTriggerStateAndHelpers);
-    return React.cloneElement(triggerComponent, {
-      triggerRef,
-      menuRefId: menuRefId.current,
-    });
+    return () =>
```

Comment:
let's add the issue number for this approach so we know what's happening. 

also might be easier to pass down the props to the consumer so we don't have to clone what they are passing? But that's for other time discussion.

Reply:
https://github.com/acl-services/paprika/issues/126